### PR TITLE
T1657

### PR DESCRIPTION
**Pull request by humphd**
... Popcorn.js]

### CODE REVIEW

File: src/util/uri.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
+/* This Source Code Form is subject to the terms of the MIT license
+ *  * If a copy of the MIT license was not distributed with this file, you can
```
Comment (secretrobotron) [inline src/util/uri.js]: This looks a bit wrong. Too many *'s?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/cf7a72cfabcb1b2/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/f5b6173f914e13e/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/cf7a72cfabcb1b2/output.txt

Total script time: 0.60 mins

Server is running on http://pr594.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/f5b6173f914e13e/output.txt

Total script time: 0.50 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/1a43bf7626e6843/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/32b7ae19b57e653/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/1a43bf7626e6843/output.txt

Total script time: 0.55 mins

Server is running on http://pr594.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/32b7ae19b57e653/output.txt

Total script time: 0.30 mins

**Pull request merged**